<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-0MS0N57" failures="1" tests="1" name="Cucumber_Automation" time="4.001" errors="0" timestamp="2023-10-05T17:32:14 IST">
  <testcase classname="com.ReqRes_Api.Parsing_xml" name="xmlParsing" time="3.93">
    <failure type="io.restassured.path.xml.exception.XmlPathException" message="Failed to parse the XML document">
      <![CDATA[io.restassured.path.xml.exception.XmlPathException: Failed to parse the XML document
at io.restassured.path.xml.XmlPath$LazyXmlParser.invoke(XmlPath.java:1039)
at io.restassured.path.xml.XmlPath.getFromPath(XmlPath.java:423)
at io.restassured.path.xml.XmlPath.get(XmlPath.java:317)
at com.ReqRes_Api.Parsing_xml.xmlParsing(Parsing_xml.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
Caused by: org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; Content is not allowed in prolog.
at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1251)
at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:637)
at groovy.xml.XmlSlurper.parse(XmlSlurper.java:199)
at groovy.xml.XmlSlurper.parse(XmlSlurper.java:252)
at groovy.xml.XmlSlurper.parseText(XmlSlurper.java:282)
at io.restassured.path.xml.XmlPath$1.method(XmlPath.java:840)
at io.restassured.path.xml.XmlPath$LazyXmlParser.invoke(XmlPath.java:1032)
... 28 more
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- xmlParsing -->
</testsuite> <!-- Cucumber_Automation -->
